operator/redisfailover/service/generator.go

```diff
@@ -160,15 +175,22 @@ func generateRedisConfigMap(rf *redisfailoverv1.RedisFailover, labels map[string
 
 func generateRedisShutdownConfigMap(rf *redisfailoverv1.RedisFailover, labels map[string]string, ownerRefs []metav1.OwnerReference) *corev1.ConfigMap {
 	name := GetRedisShutdownConfigMapName(rf)
+	port := rf.Spec.Redis.Port
 	namespace := rf.Namespace
 	rfName := strings.Replace(strings.ToUpper(rf.Name), "-", "_", -1)
 
 	labels = util.MergeLabels(labels, generateSelectorLabels(redisRoleName, rf.Name))
 	shutdownContent := fmt.Sprintf(`master=$(redis-cli -h ${RFS_%[1]v_SERVICE_HOST} -p ${RFS_%[1]v_SERVICE_PORT_SENTINEL} --csv SENTINEL get-master-addr-by-name mymaster | tr ',' ' ' | tr -d '\"' |cut -d' ' -f1)
-redis-cli SAVE
 if [ "$master" = "$(hostname -i)" ]; then
   redis-cli -h ${RFS_%[1]v_SERVICE_HOST} -p ${RFS_%[1]v_SERVICE_PORT_SENTINEL} SENTINEL failover mymaster
-fi`, rfName)
+  sleep 31
```

Comment:
What was the point of this sleep?

Given that `TerminationGracePeriodSeconds` for the redis pods is 30 I'm interpreting this as if the `SAVE` command coming after the sleep will never execute.

Is this intended?